CHANGELOG.rst

```diff
@@ -2,14 +2,141 @@
 Changelog
 =========
 
+2.7.0-alpha.0 (2024-01-05)
+==========================
+
+This is an alpha release, meaning it is not finished yet or suitable for production use.
+
+**New features**
+
+Improved backend validation robustness, mainly by validating new components:
+
+* [#4035] Added an E2E test for the file component.
+* [#72] Added validation for the following components: time, selectboxes, textarea, postcode, bsn, select, checkbox,
+  currency, signature, map, cosign, password, iban and licenseplate.
+
+Other features:
+
+* [#4031] Added a confirm popup when switching to v2 options.
+* [#4041] Improved API documents registration robustness, taking into account possible crashes when sending the documents.
```

Comment:
Sometimes you can group things by theme, e.g.:

```rst
* Submission registration:

    * Added a warning for the Objects API registration configuratoin when switching back to the legacy configuration.
    * Improved robustness of document registration in the Documents API.
```

So, couple remarks here:

* commit messages are sometimes wrong, case in point here is that it wasn't v2 that got the popup, but v1
* avoid being too detailed/technical
* provide sufficient context so that people know about which part of OF a changelog entry is

Reply:
Arrgh, did it at first and then thought it wasn't worth grouping for two features